src/config/get-config.ts

```diff
@@ -95,6 +95,14 @@ export interface Config {
    * Option to enable/disable the thanks feature.
    */
   THANK_OPTION: boolean;
+  /**
+   * Announcement message ID for Halloween event
+   */
+  HALLOWEEN_ANNOUNCEMENT: string;
+  /**
+   * Role for Halloween event.
+   */
+  HALLOWEEN_ROLE: string;
```

Comment:
Update the comment with how the role is to be used

Reply:
Done.